### ISSUE TITLE
Ability to customize Ambassador configuration

### ISSUE THREAD

**Issue by cliveseldon** [SeldonIO/seldon-core#120]
At present the REST and gRPC Ambassador configuration is set by the cluster-manager.
We should allow the ability to customize this config.

**Comment by codebreach** [comment]
copying from #196 

As per #211 the timeouts on REST and gRPC can be configured by using annotations.

However the Ambassador config is not updated to reflect this.
`timeout_ms` is not set on the service generated even though we pass the annotations

```yaml
kind: Service
metadata:
  annotations:
    getambassador.io/config: |
      ---
      apiVersion: ambassador/v0
      kind:  Mapping
      name:  seldon_dpa_rest_mapping
      prefix: /seldon/dpa/
      service: dpa:8000
      ---
      apiVersion: ambassador/v0
      kind:  Mapping
      name:  dpa_grpc_mapping
      grpc: true
      prefix: /seldon.protos.Seldon/
      rewrite: /seldon.protos.Seldon/
      headers:
        seldon: dpa
      service: dpa:5001
```
it should include
```yaml
      timeout_ms: VALUE_FROM_ANNOTATION
```

this is actually fixed by #250

**Comment by cliveseldon** [comment]
You can try with 0.2.4-SNAPSHOT build from fix in #250 
Can you test and report back?

**Comment by codebreach** [comment]
Verified :)
